.github/workflows/commit_message.yml

```diff
@@ -40,4 +41,4 @@ jobs:
             fi
           fi
           echo "message=$RUN" >> $GITHUB_OUTPUT
-          echo github.ref ${{ github.ref }}
+          echo github.ref $GITHUB_REF
```

Comment:
These aren't valid alerts, as noted in https://github.com/scipy/scipy/issues/22966#issuecomment-2888253340, however the change itself here is perfectly okay and a bit easier to read, so let's keep it.